packages/babel-preset-default/index.js

```diff
@@ -58,12 +58,14 @@ module.exports = function( api ) {
 				require.resolve( '@wordpress/babel-plugin-import-jsx-pragma' ),
 				{
 					scopeVariable: 'createElement',
+					scopeVariableFrag: 'Fragment',
 					source: '@wordpress/element',
 					isDefault: false,
 				},
 			],
 			[ require.resolve( '@babel/plugin-transform-react-jsx' ), {
 				pragma: 'createElement',
+				pragmaFrag: 'Fragment',
```

Comment:
Re: Abbreviations. I suppose part of the purpose is in inheriting the Babel option by the same name?

https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragmafrag

A bit unfortunate naming. I guess I'd rather be consistent than avoid the abbreviation, though.